modules/highgui/src/window_QT.cpp

```diff
@@ -3308,40 +3314,39 @@ void OpenGlViewPort::paintGL()
 void OpenGlViewPort::wheelEvent(QWheelEvent* evnt)
 {
     icvmouseEvent((QMouseEvent *)evnt, mouse_wheel);
-    QGLWidget::wheelEvent(evnt);
+    QWidget::wheelEvent(evnt);
```

Comment:
`OpenCVQtWidgetBase::` ?

Reply:
Rescoped from QWidget to OpenCVQtWidgetBase